### PR TITLE
Feature: Updated for WithdrawalCaps and SupportedCollateral support

### PR DESCRIPTION
**Pull request by ignaty8**

### CODE REVIEW

File: packages/hubble-idl/src/borrowing.json [hunk 1]
```diff
@@ -3040,12 +3411,41 @@
               ]
             }
           },
+          {
```
Comment (elliotkennedy) [inline packages/hubble-idl/src/borrowing.json:3414]: Add these 3 new fields to `BorrowingMarketState.ts`

```
          {
            "name": "liquidationPenaltyPoolBased",
            "type": "u64"
          },
          {
            "name": "liquidationPenaltyCashBased",
            "type": "u64"
          },
          {
            "name": "minCashBasedLiquidationRepayment",
            "type": "u64"
          },
```

File: packages/hubble-idl/src/borrowing.json [hunk 2]
```diff
@@ -3720,6 +4195,48 @@
         ]
       }
     },
+    {
+      "name": "WithdrawalCaps",
+      "type": {
+        "kind": "struct",
+        "fields": [
+          {
+            "name": "configCapacity",
```
Comment (elliotkennedy) [inline packages/hubble-idl/src/borrowing.json:4204]: Are these `i64` values being mapped to `BNs` and then to `Decimals` correctly?

**Review by elliotkennedy — CHANGES_REQUESTED** [review]
Looks good - just one comment

File: packages/hubble-idl/src/borrowing.json (hunk 2 above)
Comment (ignaty8) [reply packages/hubble-idl/src/borrowing.json:4204]: Yup, they are!

File: packages/hubble-sdk/src/models/BorrowingMarketState.ts [hunk 3]
```diff
@@ -26,7 +29,13 @@ export type BorrowingMarketState = {
   stablecoinRewardPerToken: Decimal;
   totalStakeSnapshot: Decimal;
   borrowedStablecoinSnapshot: Decimal;
+  supportedCollaterals: SupportedCollateral[];
   marketMcr: Decimal;
+  liquidation_penalty_pool_based: Decimal;
```
Comment (elliotkennedy) [inline packages/hubble-sdk/src/models/BorrowingMarketState.ts:34]: Why is this underscored? Does it work? Can you test it please

**Review by elliotkennedy — APPROVED** [review]

**Pull request merged**